### ISSUE TITLE
[FEATURE] Upgrade the DSharpPlus package to v4 RC

### ISSUE THREAD

**Issue by anf** [anf/BotANF#11]
**Is your feature request related to a problem? Please describe.**
[v4 of DSharpPlus](https://www.nuget.org/packages/DSharpPlus/4.0.0-rc1) is in RC which technically means that it's stable enough to run well, at least better than the current preview package which is not accessible the usual way.

**Describe the solution you'd like**
Change the package to v4.0.0 RC-1.

**Additional context**
Doing so will help us continue our github actions as it would be accessible by nuget. You can do some other stuff to get that done without switching, but eh.

TL;DR:
- It would resume github actions
- It would be more stable, as in trusted not to break

**Comment by tomlin7** [comment]
v4 is stable

**Comment by anf** [comment]
Not necessarily, but we need to look more into this, for example pros and cons.
Plans are most likely going to be uprading BotANF to the RC version.

**Comment by billydevyt** [comment]
Let's do the project after 2 years, d#+ will be stable then, pog

**Comment by anf** [comment]
Done with commit [dev/b1ed383](https://github.com/ANF-Studios/BotANF/commit/b1ed38300445e8b1df8230c62a73b8d781460684)
